test/tools/jsontests/TransactionTests.cpp

```diff
@@ -38,134 +39,167 @@ namespace fs = boost::filesystem;
 
 namespace dev {  namespace test {
 
+mObject getExpectSection(mValue const& _expect, eth::Network _network)
+{
+	std::vector<mObject> objVector;
+	BOOST_REQUIRE(_expect.type() == json_spirit::array_type);
+	for (auto const& value: _expect.get_array())
+	{
+		BOOST_REQUIRE(value.type() == json_spirit::obj_type);
+		mObject obj;
+		obj = value.get_obj();
+		BOOST_REQUIRE_MESSAGE(obj.count("network"), "network section not set in expect section!");
+		vector<string> networks;
+		ImportTest::parseJsonStrValueIntoVector(obj["network"], networks);
+		BOOST_CHECK_MESSAGE(networks.size() > 0, TestOutputHelper::get().testName() + " Network array not set!");
+		ImportTest::checkAllowedNetwork(networks);
+
+		if (test::inArray(networks, test::netIdToString(_network)) || test::inArray(networks, string("ALL")))
+			objVector.push_back(obj);
+	}
+	BOOST_REQUIRE_MESSAGE(objVector.size() == 1,"Expect network should occur once in expect section of transaction test filler! (" + test::netIdToString(_network) + ")");
+	return objVector.at(0);
+}
+
+json_spirit::mObject FillTransactionTest(json_spirit::mObject const& _o)
+{
+	mObject out;
+	BOOST_REQUIRE_MESSAGE(_o.count("transaction") > 0, "transaction section not found! " + TestOutputHelper::get().testFileName());
+	BOOST_REQUIRE_MESSAGE(_o.count("expect") > 0, "expect section not found! " + TestOutputHelper::get().testFileName());
+
+	mObject tObj = _o.at("transaction").get_obj();
+	RLPStream rlpStream = createRLPStreamFromTransactionFields(tObj);
+	out["rlp"] = toHexPrefixed(rlpStream.out());
+
+	// Theoretical block for transaction check
+	BlockHeader bh;
+	bh.setNumber(1);	//Seal engine below enables network rules from block 0
+	bh.setGasLimit(u256("0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff"));
+
+	mValue expectObj = _o.at("expect");
+	for (auto const network: test::getNetworks())
+	{
+		unique_ptr<SealEngineFace> se(ChainParams(genesisInfo(network)).createSealEngine());
+		bool onConstantinople = (network == eth::Network::ConstantinopleTest);
```

Comment:
This `==` should be replaced with `>=` as well.

Reply:
this wont work. there are other networks (configurations) larger than this. it does not mean what you expect it to mean. I think it is better to select networks more explicitly here

Reply:
Then, use a function like `hasZeroSignature(network)`.

Reply:
this function is already used.

Reply:
Why not here?